hack/run-test-must-gather-e2e.sh

```
@@ -10,8 +10,85 @@ if ! which tput &> /dev/null 2>&1 || [[ $(tput -T$TERM colors) -lt 8 ]]; then
+while [[ $# -gt 0 ]]; do
+	case "$1" in
+		--no-color)
+			NO_COLOR="-ginkgo.no-color"
+			shift
+			;;
+		--dry-run)
+			DRY_RUN="true"
+			shift
+			;;
+		--focus)
+			FOCUS="-ginkgo.focus='$2'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)